csxtools/utils.py

```diff
@@ -12,7 +12,8 @@
 
 
 def get_fastccd_images(light_header, dark_headers=None,
```

Comment:
@vivekthampy I think this is a good idea, but concerned about adding such complexity. You say that this "significantly" reduces processing .... but with the new c-code it takes 2-3 seconds to process the stacks and do the background mean so I am a little confused about the "significant" comment.